packages/ibm-products/src/components/Decorator/constants.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const getMagnitude = (score, thresholds) => {
+  if (typeof score === 'number') {
+    if (score <= thresholds[0]) {
+      return 'Benign';
+    }
+    if (score < thresholds[1]) {
+      return 'Low';
+    }
+    if (score < thresholds[2]) {
+      return 'Medium';
+    }
+    if (score < thresholds[3]) {
+      return 'High';
+    }
+    if (score >= thresholds[3]) {
+      return 'Critical';
+    }
+  }
+
+  return 'Unknown';
+};
+
+const truncate = (inputText, maxLength, front, back) => {
+  let truncatedText = inputText;
```

Comment:
whenever possible we should try to avoid unnecessary mutation. we can simplify this to just be something like
```
if (thing) return ${inputText.substring.otherstuff
return inputText;
```

Reply:
Done.